src/Smalot/PdfParser/RawData/RawDataParser.php

```diff
@@ -470,8 +470,8 @@ protected function getIndirectObject($pdfData, $xref, $obj_ref, $offset = 0, $de
             throw new Exception('Invalid object reference for $obj.');
         }
         $objref = $obj[0].' '.$obj[1].' obj';
-        // ignore leading zeros
-        $offset += strspn($pdfData, '0', $offset);
+        // ignore newlines & leading zeros
+        $offset += strspn($pdfData, "0\r\n", $offset);
```

Comment:
In the link you posted they also check whitespaces. But here you only ignore newlines and leading zeros.

https://github.com/mozilla/pdf.js/blob/v2.5.207/src/core/obj.js#L1504

Reply:
And can you tell me, if there are any changes in future PDF versions related to this?

Reply:
added ignoring of whitespaces (see ISO 32000-1 > 7.2.2 Character Set > Table 1) which is revised version for PDF-1.7